src/test/gen_list.ts

```diff
@@ -0,0 +1,79 @@
+const fs = require('fs')
+import { getOpenseaCollectionAddresses } from "../../script/opesea_contrats"
+
+import {
+    Ethereum,
+    getChainAssetsPath,
+    readDirSync,
+    readFileSync,
+    isChainWhitelistExistSync,
+    isChainBlacklistExistSync,
+    getChainWhitelistPath,
+    getChainBlacklistPath,
+    writeFileSync,
+    sortDesc,
+    getUnique,
+    mapList
+} from './helpers'
+
+// Current chains where assets tokens supported by Trust Wallet
+// TODO auto create list based on assets folder presence
+const chainsSupportedWhiteAndBlacklist = [
```

Comment:
did you consider having shorter names for variables?

Reply:
can you remove comments? as you can explain with name variable

Reply:
Do we need it? Is it possible just to iterate over blockchains directory and process those containing assets directory inside?

Reply:
https://github.com/trustwallet/assets/pull/502/commits/9e8ab2a3deb361b38deca959c91fbd5e46117478

Reply:
Initial idea was to have list in case someone adds assets to new chain without tests